lib/liquid/standardfilters.rb

```diff
@@ -370,7 +370,10 @@ def join(glue)
       end
 
       def concat(args)
-        to_a.concat args
+        unless args.respond_to?(:to_ary)
```

Comment:
No, we don't call `args.to_a` below, and Array#concat uses `.to_ary`.  Also, `nil.to_a` returns `[]` and we don't want that implicit conversion.